app/views/api/v1/orders/show.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.instance do
+  json.user @order.user_id
+  json.status @order.status
+
+  if @order.menu_items
+    json.items @order.menu_items do |item|
+      json.id item.id
+      json.item item.name
+      json.price item.price.to_f
+    end
+  else
+    json.items '"no items"'
+  end
+
+end
```

Comment:
I get your point. But at the same time i end up with too much chaining to get to what I need on the front end. When i request an `order` i expect to get an order, i'm likely going to store the response in an `order` object. Doing `order.instance` to get to the order details doesn't feel natural anymore